apps/remix-ide/src/app/files/dgitProvider.js

```diff
@@ -38,6 +38,13 @@ class DGitProvider extends Plugin {
       protocol: 'https',
       ipfsurl: 'https://ipfs.io/ipfs/'
     }
+    this.remixIPFS = {
```

Comment:
`ipfsconfig` already uses the same URL, is that wanted?

Reply:
ipfsconfig is set by the dgit client. so it can be anything. so it's not a duplicate per se.